ci/docker-entrypoint.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 rm -rf build
-mkdir build && cd build
-cmake .. && make && make python
-make check
+mkdir -p build
+cd build || exit 1
+cmake .. && make -j32 && make python -j32
+make check -j32
```

Comment:
I like that the build is much faster with `-j32`, but I don't think it has an effect for `make check`.

Reply:
I know. I just kept it anyway, assuming it might do something under the hood :)
It's not causing any issues, I think it's okay to keep it.